Libraries/Text/Text.js

```diff
@@ -153,6 +153,15 @@ const Text: React.AbstractComponent<
       : processColor(restProps.selectionColor);
 
   let style = restProps.style;
+
+  if (style && style.verticalAlign !== undefined) {
```

Comment:
This will break unless styles are flattened into an object. But can we look into moving this feature into the code that handles adding styles internally to the native nodes, rather than doing it in the JS for each component?

Reply:
Ohh @necolas that's true. I'm no expert on the react-native code base but from my understanding there isn't a file like [createReactDOMStyle](https://github.com/gabrieldonadel/react-native-web/blob/dee258ae02188befca4192ea502d4fbebc478341/packages/react-native-web/src/exports/StyleSheet/compiler/createReactDOMStyle.js#L110) on react-native-web that handles and parses styles for all components, but maybe I'm missing something. I can also add support for this prop natively on android if you prefer, something similar to https://github.com/facebook/react-native/blob/c34659a5d3ef24e6a0ace2a5cac242487225fed0/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextAnchorViewManager.java#L78
But this would also require us to handle each component individually.

Reply:
I see. Well if styles are always flattened somewhere in RN before being set on native, maybe it doesn't matter if we add it to the JS components that don't already do it. Image for example is already flattening styles anyway

Reply:
Added the `flattenStyle` to both files